nservicebus/upgrades/5to6.md

```diff
@@ -39,19 +39,43 @@ The endpoint name is used as a logical identifier when sending or receiving mess
 
 ## Message handlers
 
-The handler method on `IHandleMessages<T>` now returns a Task. In order to leverage async code, add the `async` keyword to the handler method and use `await` for async methods. In order to convert the synchronous code add `return Task.FromResult(0);` to the end of the handler methods.
+The handler method on `IHandleMessages<T>` now returns a Task. In order to leverage async code, add the `async` keyword to the handler method and use `await` for async methods. In order to convert the synchronous code add `return Task.FromResult(0);` or `return Task.CompletedTask` (.NET 4.6 and higher) to the end of the handler methods.
 
 WARNING: Do not `return null` from the message handlers. A `null` will result in an Exception.
 
 snippet:5to6-messagehandler
 
-
 ### Bus Send and Receive
 
 There is also a change in the parameters, giving access to the `IMessageHandlerContext`, which provides the methods that used to be called from `IBus`. Use the `IMessageHandlerContext` to send and publish messages.
 
 snippet:5to6-bus-send-publish
 
+### Migrate existing handlers
+
+A simplified sample how an existing handler could look like.
```

Comment:
How about:
In previous versions of NServiceBus, a typical handler class looks like the below: